lib/modules/platform/bitbucket-server/index.ts

```diff
@@ -109,9 +110,9 @@ export async function initPlatform({
   };
   try {
     let bitbucketServerVersion: string;
-    // istanbul ignore if: experimental feature
-    if (process.env.RENOVATE_X_PLATFORM_VERSION) {
-      bitbucketServerVersion = process.env.RENOVATE_X_PLATFORM_VERSION;
+    const platformVersion = GlobalConfig.get('platformVersion');
+    if (platformVersion) {
+      bitbucketServerVersion = platformVersion;
```

Comment:
pass this as an initPlatform() parameter instead of reading from GlobalConfig.get()